ext/pdo/pdo_stmt.c

```diff
@@ -2405,6 +2406,7 @@ static zend_object *pdo_row_new(zend_class_entry *ce)
 {
 	pdo_row_t *row = zend_object_alloc(sizeof(pdo_row_t), ce);
 	zend_object_std_init(&row->std, ce);
+	object_properties_init(&row->std, pdo_row_ce);
```

Comment:
question ; would `ce` just work here ?

Reply:
Yep it would, in this case it's always the same. But it's cleaner to use `ce`. This is a copy-pasta leftover. Thanks for noticing